wayback-webapp/src/main/webapp/WEB-INF/ProxyReplay.xml

```diff
@@ -6,9 +6,13 @@
 
   <bean id="identityhttpheaderprocessor" class="org.archive.wayback.replay.IdentityHttpHeaderProcessor" />
 
-  <bean id="proxytransparentreplayrenderer" class="org.archive.wayback.replay.TransparentReplayRenderer">
+  <!-- use org.archive.wayback.replay.HttpsRedirectRewriteReplayRendeder instead of
+  org.archive.wayback.replay.TransparentReplayRendeder to force https url rewrite -->
+  <bean id="proxytransparentreplayrenderer" class="org.archive.wayback.replay.TransparentReplayRendeder">
```

Comment:
If this remains the default,
 class="org.archive.wayback.replay.TransparentReplayRendeder"
should be
 class="org.archive.wayback.replay.TransparentReplayRende**rer**"